code/style.css

```diff
@@ -2,6 +2,359 @@
 in this case it is the div with the answer class next to the div with the active class.
 this can be used for the FAQ-accordion. read more here: https://www.w3schools.com/css/css_combinators.asp */
 
+body {
+  margin: 0;
+  padding: 0;
+  font-family: 'Open Sans', sans-serif;
+  font-size: 18px;
+  
+  background-image: url(./images/annie-spratt-vs6a4EHj_Ro-unsplash\ \(1\).jpg);
+  background-size: cover;
+  background-repeat: no-repeat;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  background-position: right;
+  position: relative;
+   
+}
+
+
+h1 {
+  font-family: 'Major Mono Display', monospace;
+  font-size: 60px;
+  color: white;
+  text-align: center;
+  text-shadow: 2px 2px 7px rgba(0, 0, 0, 0.5);;
+  margin-bottom: 0;
+}
+
+h2 {
+  font-family: 'Major Mono Display', monospace;
+  font-size: 50px;
+  color: white;
+  text-align: center;
+  text-shadow: 1px 1px 5px rgba(0, 0, 0, 0.5);;
+}
+
+h3 {
+  font-family: 'Major Mono Display', monospace;
+  font-size: 30px;
+  color: #232425;
+ 
+}
+
+
+header {
+  min-height: 200px;
+  margin-top: 10%;
+  margin-bottom: 10%;
+
+}
+
+.headertext {
+  color: #ffff;
+  font-size: 20px;
+  text-align: center;
+  text-shadow: 2px 2px 7px rgb(0, 0, 0);
+  margin-left: 30px;
+  margin-right: 30px;
+
+}
+
+
+/* Form styling starts here */
+
+.wrap-form {
+  display: flex;
+  width: 400px;
+  margin: 30px auto;
+
+}
+
+form {
+  background-color: rgba(228, 218, 244, 0.85);
+  border: 1px solid;
+  border-radius: 8px;
+  padding: 20px;
+  box-shadow: 5px 5px 15px 5px rgba(0,0,0,0.5);
+
+}
+
+label {
+  display: block;
+  font-size: 16px;
+  /* font-family: 'Open Sans', sans-serif; */
+  color: #232425;
+
+}
+
+.radiolabel {
+  display: inline-block;
+
+}
+ 
+
+input {
+  display: block;
+  font-size: 20px;
+  padding: 5px;
+	margin-bottom: 20px;
+  border-radius: 5px;
+
+}
+
+.radiobtn {
+  display: inline-block;
+  margin-bottom: 10px;
+  font-size: 30px;
+
+}
+
+button {
+  background-color: #ffff;
+  color: #232425;
+  padding: 10px 20px;
+  margin-bottom: 20px;
+  border-radius: 5px;
+  text-align: center;
+  text-decoration: none;
+  display: inline-block;
+  font-size: 18px;
+
+}
+
+
+/* Accordion styling starts here */
+
+.wrap-accordion {
+  display: flex;
```

Comment:
I don't really understand what you are flexing in "wrap-accordion" or what the intention is. You are flexing the body as well, and that's correct if you want to flex the accordion and the form.